keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -235,51 +452,101 @@ _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
 
-  TBD for beta.
+The primary method of rollout failure is due to configuration issues. When
```

Comment:
> Can the feature be disabled once it has been enabled (i.e. can we roll back
  the enablement)

That section should at least mention the guide that descibed how upgrades/downgrades should be done
[already mentioned somewhere above]


> Are there any tests for feature enablement/disablement?

Were they added? Please update this answer.

Reply:
ack, will update